lib/shoulda/matchers/integrations/test_frameworks.rb

```diff
@@ -1,15 +1,15 @@
-require 'shoulda/matchers/integrations/test_frameworks/active_support_test_case'
-require 'shoulda/matchers/integrations/test_frameworks/minitest_4'
-require 'shoulda/matchers/integrations/test_frameworks/minitest_5'
 require 'shoulda/matchers/integrations/test_frameworks/missing_test_framework'
 require 'shoulda/matchers/integrations/test_frameworks/rspec'
-require 'shoulda/matchers/integrations/test_frameworks/test_unit'
 
 module Shoulda
   module Matchers
     module Integrations
       # @private
       module TestFrameworks
+        autoload :ActiveSupportTestCase, 'shoulda/matchers/integrations/test_frameworks/active_support_test_case'
```

Comment:
I think this is a case where these files can't be autoloaded, because there is a bit of magic here where upon being loaded, they will register themselves as possible test frameworks. This makes it possible to say e.g.

``` ruby
Shoulda::Matchers.configure do |config|
  config.integrate do |with|
    with.test_framework :rspec   # or whatever
  end
end
```

If the appropriate file that corresponds to the symbol you're setting isn't loaded then it doesn't know what to do and the matchers will never be integrated into the appropriate test framework. Perhaps there is another way to accomplish this to where we can autoload them, but that's how they work now. This may be why the tests are failing.

Reply:
I made a silly mistake. I was only running unit tests.

I think the best strategy is to create a new branch of the master and make the changes little by little to see what went wrong.

This time I'll run all the tests at every change! :sweat_smile: 

Thanks for the info!

Reply:
Done! Tests fixed!